internal/metrics/metrics.go

```diff
@@ -282,14 +289,17 @@ func (p *PrometheusReporter) Info(strongDucklingVersion string) {
 }
 
 func (r *tcpChecker) ReportPortCheck(report tcpchecker.Report) {
+	labelValues := []string{report.Name, report.Address, fmt.Sprintf("%d", report.Port)}
```

Comment:
of cause we also need a (pseudo syntax here):
```suggestion
	labelValues := []string{report.Name, report.Address, fmt.Sprintf("%d", report.Port), report.Open ? "open" : "closed"}
```